contracts/mocks/docs/ERC4626Fees.sol

```diff
@@ -64,27 +69,33 @@ abstract contract ERC4626Fees is ERC4626 {
         }
     }
 
-    function _entryFeeBasePoint() internal view virtual returns (uint256) {
-        return 0;
+    // === Fee configuration === ///
+
+    function _entryFeePCM() internal view virtual returns (uint256) {
+        return 0; // i.e. 1_000 for 1%
```

Comment:
The use of `i.e.` in these comments is not right. `0 // i.e. 1000` means "0, equivalently 1000".

We should write something like `// replace with e.g. 1_000 for 1%`.

Reply:
TIL `i.e.` is not `in example`

Solved.